### DISCUSSION TITLE
linux install instructions unclear

### DISCUSSION THREAD

**Discussion by NicTanghe** [CastagnaIT/plugin.video.netflix#1469] [Q&A] [answered]
Hello,

 the only reason I would use Kodi i s to watch Netflix in not 720p

There is no Netflix thing anywhere except a screenshot of the UI in the add-on.

The installation instructions read to run.

```
python NFAuthenticationKey.py 
```

However the script is not in the repository and nowhere to be found .

**Comment by CastagnaIT** [comment, marked as answer]
I don't see the problem
just click on "DOWNLOAD:" link
